packages-content-model/roosterjs-content-model-core/lib/editor/StandaloneEditor.ts

```diff
@@ -365,6 +366,20 @@ export class StandaloneEditor implements IStandaloneEditor {
         return this.getCore().trustedHTMLHandler;
     }
 
+    /**
+     * Get the scroll container of the editor
+     */
+    getScrollContainer(): HTMLElement {
+        return this.getCore().domEvent.scrollContainer;
+    }
+
+    /**
+     * Retrieves the rect of the visible viewport of the editor.
+     */
+    getVisibleViewport(): Rect | null {
+        return this.getCore().api.getVisibleViewport(this.getCore());
+    }
+
```

Comment:
please create some test cases for these two functions in StandaloneEditorTest.ts